### PR TITLE
Bugfix: pupil login

### PR DESCRIPTION
**Pull request by jon-shipley**

### CODE REVIEW

File: admin/services/data-access/sql.service.js [hunk 1]
```diff
@@ -178,6 +178,8 @@ sqlService.adminSchema = '[mtc_admin]'
  * @return {Promise<*>}
  */
 sqlService.query = (sql, params = []) => {
+  winston.debug(`sql.service.query(): ${sql}`)
+  winston.debug('sql.service.query(): Params ', R.compose(R.map(R.pick(['name', 'value'])))(params))
```
Comment (GuyHarwood) [inline admin/services/data-access/sql.service.js]: 🥇

Comment (GeorgeChatzigiannis) [reply admin/services/data-access/sql.service.js]: Are you 100% sure the compose is needed?

Comment (jon-shipley) [reply admin/services/data-access/sql.service.js]: @GeorgeChatzigiannis - I'm sure someone can find a way to remove `Compose`.

Comment (GeorgeChatzigiannis) [reply admin/services/data-access/sql.service.js]: Well the output seems to be the same with or without it for me.

**Pull request merged**